src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -86,8 +86,7 @@ export class MemberListComponent implements OnChanges, OnDestroy {
     private messageService: MessageService,
   ) {
     this.dataFetching.pipe(
-      switchMap(params => this.getData(params.groupId, params.filter, params.sort)),
-      mapToFetchState(),
+      switchMap(params => this.getData(params.groupId, params.filter, params.sort).pipe(mapToFetchState())),
```

Comment:
Why?

Reply:
Have you tried the resetter at the end?

Reply:
I tried to pass the `dataFetching` Subjet as an observable but le table is stuck on loading, so I kept it like it is.